### PR TITLE
beta docs for offer

### PR DESCRIPTION
**Pull request by shalehaha**

### CODE REVIEW

File: beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:20]: Grammar: ...select the configuration **that** needs to be updated.

Edge Configurations is at the same level as Properties. Why do we need to navigate to the mobile property first? Isn't it the inlet that specifies the source, eg mobile app?

File: beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:22]: Fix: Enabled -> Enable

File: beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md
Comment (swarna04) [reply beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:20]: Did you mean select the right IMS org first?

File: beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:26]: Typo Decisioniong -> Decisioning

File: beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:117]: typo `convinient`

File: beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:134]: typo `seperate`, `retrives`, `recommand`, `rerieve`, `reder`

File: beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:189]: Fix: ...triggered by the aforementioned...

Comment (swarna04) [reply beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md:189]: Typo: `respsonses`

File: beta/experience-platform-extension/offerdecisioning/offer-api.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-api.md:132]: typo `wich`

File: beta/experience-platform-extension/offerdecisioning/offer-api.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-api.md:167]: Fix: An enum indicating...

File: beta/experience-platform-extension/offerdecisioning/offer-api.md
Comment (swarna04) [inline beta/experience-platform-extension/offerdecisioning/offer-api.md:176]: Can we not just write 
```
public enum OfferType: Int {
    case unknown
    case json
    case text
    case html
    case image
}
```
Swift will automatically assign 0 to unknown and count upwards from there.

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: beta/experience-platform-extension/offerdecisioning/offer-api.md
Comment (shalehaha) [reply beta/experience-platform-extension/offerdecisioning/offer-api.md:176]: this class needs to be objective-c compatible, i haven't looked much into this but followed the way how we define enums, I guess the explicit int is needed to support obj-c. I didn't include all that tech detail in the docs .

**Review by imironchuk — APPROVED** [review]
Looks good to me

File: beta/experience-platform-extension/offerdecisioning/offer-api.md
Comment (swarna04) [reply beta/experience-platform-extension/offerdecisioning/offer-api.md:176]: Yes that's correct. Enum cases in swift don't have integer values set by default unlike obj-C . So once we define an enum to be of a particular type, say Int we do not need to assign a raw value to each case (Implicit assignment). Also the first case starts with 0 (unless specified) and the rest of the cases count upwards. Anyways, I'll leave it up to you.

**Review by swarna04 — CHANGES_REQUESTED** [review]

**Comment by swarna04** [comment]
Suggested a few more minor edits before we merge this @shalehaha

**Pull request merged**
